dev/teamcity/dist-assets-tc

```diff
@@ -17,7 +17,8 @@ echo "##teamcity[progressFinish 'asset validation and tests']"
 
 echo "##teamcity[progressStart 'asset dist']"
 set -x
-./grunt-tc compile-assets emitAbTestInfo
+make ARGS="--stack --no-color" compile
```

Comment:
passing these `grunt` args in here is temporary way of suppressing the colours etc in teamcity while the discreet compilation steps still use grunt